src/core/net/ip6.hpp

```diff
@@ -379,7 +379,7 @@ class Ip6 : public InstanceLocator, private NonCopyable
     static constexpr uint8_t kDefaultHopLimit      = OPENTHREAD_CONFIG_IP6_HOP_LIMIT_DEFAULT;
     static constexpr uint8_t kIp6ReassemblyTimeout = OPENTHREAD_CONFIG_IP6_REASSEMBLY_TIMEOUT;
 
-    static constexpr uint16_t kMinimalMtu = 1280;
```

Comment:
The value 1280 is specified by [RFC 8200 Section 5](https://www.rfc-editor.org/rfc/rfc8200#section-5).